packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -83,50 +86,62 @@ export default function Classifier({
   const classifierStore = useStore({
     authClient,
     client,
-    initialState: {
-      projects: {
-        active: project.id,
-        resources: {
-          [project.id]: project
-        }
-      }
-    }
+    initialState: {}
   })
 
-  const {
-    classifications,
-    projects,
-    subjects,
-    userProjectPreferences,
-    workflows
-  } = classifierStore
+  const [loaded, setLoaded] = useState(false)
 
-  useEffect(function onMount() {
+  async function onMount() {
+    try {
+      const storageKey = `fem-classifier-${project.id}`
+      await persist(storageKey, classifierStore, {
+        storage: asyncSessionStorage,
```

Comment:
✅  Dev Note: this where we inform `mst-persist` to use window.sessionStorage. By default, mst-persist uses _localStorage,_ which has ramifications on long-term storage